storage/innobase/dict/dict0load.cc

```diff
@@ -508,6 +524,35 @@ dict_process_sys_virtual_rec(
 	return(err_msg);
 }
 
+/** This function parses a SYS_COLUMNS_ADDED record and extracts added column
+information
+@param[in,out]	heap		heap memory
+@param[in]	rec		current SYS_COLUMNS_ADDED rec
+@param[in]	index	cluster index of SYS_COLUMNS_ADDED 
+@param[in,out]	table_id	table id
+@param[in,out]	pos				column position
+@param[in,out]	def_val		default value of column	
+@return error message, or NULL on success */
+const char*
+dict_process_sys_columns_added_rec(
+	mem_heap_t*	heap,
+	const rec_t*	rec,
+	dict_index_t*	index,
+	table_id_t*	table_id,
+	ulint*			pos,
+	char**			def_val,
+	ulint*			def_val_len)
+{
+	const char*	err_msg;
+
+	/* Parse the record, and get "dict_col_def_t" struct filled */
+	err_msg = dict_load_column_added_low(NULL, heap, rec, index, 
+					table_id, pos, def_val, def_val_len);
+
+	return(err_msg);
+}
+
+
```

Comment:
Can we remove this function wrapper and just call `dict_load_column_added_low()` directly?
In wrappers like this, I think that it would be better to keep the function parameters in the same order, and add any new parameters to the end. Now the added parameter is the first, and this could generate a lot of code to shuffle the parameter registers.